R/PackageFunctionReporter.R

```diff
@@ -154,7 +155,7 @@ FunctionReporter <- R6::R6Class(
             # Set Graph to Color By Coverage
             private$set_plot_node_color_scheme(
                 field = "coverageRatio"
-                , pallete = c("red", "green")
+                , palette = c("red", "green")
```

Comment:
these `palette` changes are really distracting. It's cool for this but in the future could you please make stuff like that a separate PR?

Reply:
Sorry, will do!